src/app.py

```diff
@@ -856,15 +856,15 @@ def create_entity(entity_type):
 
     # Validate request json against the yaml schema
     try:
-        schema_manager.validate_json_data_against_schema(json_data_dict, normalized_entity_type)
+        schema_manager.validate_json_data_against_schema('ENTITIES', json_data_dict, normalized_entity_type)
```

Comment:
@maxsibilla I don't understand why we would hardcode this 'ENTITIES' key everywhere when this `schema_manager.validate_json_data_against_schema()` being called, instead of just hardcoding once in the method definition.